net/openvswitch/files/openvswitch.config

```diff
@@ -6,3 +6,8 @@ config ovn_northd north
 
 config ovn_controller controller
 	option disabled 1
+
+config ovs_bridge
+	option name 'my-bridge'
+	option controller '192.168.0.1'
+	option controller_port '6653' # optional
```

Comment:
I think `controller` alone should suffice.  I suppose that's what regular ovs users are familiar with.  I prefer for the moment the init script do not prefix it with "tcp:" but we can do that later if that is needed.  The point is try to stick with what people are already familiar with and less burden of new trivial things.

Reply:
Another thing is that this section will need a `disabled` option.  Value of that should be 1 in the sample config.

Reply:
> I think `controller` alone should suffice. I suppose that's what regular ovs users are familiar with. I prefer for the moment the init script do not prefix it with "tcp:" but we can do that later if that is needed. The point is try to stick with what people are already familiar with and less burden of new trivial things.

The transport prefix (e.g. "tcp:") is mandatory, so
I've added this to the sample config. If we omit the 
transport protocol, we get an error like :
"ovs|00002|vsctl|WARN|target type "1.2.3.4" is possibly erroneous".